Tests/Mapping/Loader/ReflectionLoaderTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Tests\Doctrine\Mapping\Loader;
+
+use Dunglas\ApiBundle\Mapping\Factory\AttributeMetadataFactory;
+use Dunglas\ApiBundle\Mapping\Loader\ReflectionLoader;
+
+class MyTestClass {
+  private $SIRET;
+  public function getSIRET() { return $this->SIRET; }
+  public function setSIRET($SIRET) { $this->SIRET = $SIRET; }
+}
+
+class ReflectionLoaderTest extends \PHPUnit_Framework_TestCase
+{
+  
+    public function testUppercaseMethods() 
+    {
+        $testClass = new MyTestClass();
+
+        $reflectionClassProphecy = $this->prophesize('ReflectionClass');
```

Comment:
Duno, I used [this](https://github.com/dunglas/DunglasApiBundle/blob/master/Tests/Mapping/Loader/AnnotationLoaderTest.php) as an example and I probably got lost trying to understand what should call whom. It's a bit hard to write this kind of tests when you're not familiar with the ApiBundle internals, in my opinion.